src/main/java/org/fisco/bcos/channel/client/CallContract.java

```diff
@@ -0,0 +1,278 @@
+            transactionReceipt.setMessage(StatusCode.getStatusMessage(status));
+
+        } catch (Exception e) {
+            transactionReceipt.setStatus(StatusCode.ExceptionCatched);
+            transactionReceipt.setMessage(e.getMessage());
+            transactionReceipt.setOutput("0x");
+        }
+        return transactionReceipt;
+    }
+
+    public TransactionReceipt sendTransaction(
+            BigInteger gasPrice,
+            BigInteger gasLimit,
+            String contractAddress,
+            String funcName,
+            Type... args) {
+        final Function function =
+                new Function(
+                        funcName,
+                        Arrays.<Type>asList(args),
+                        Collections.<TypeReference<?>>emptyList());
+        TransactionReceipt transactionReceipt = new TransactionReceipt();
+        try {
+            ExecuteTransaction executeTransaction =
+                    new ExecuteTransaction(contractAddress, web3j, credentials, gasPrice, gasLimit);
+            transactionReceipt = executeTransaction.send(function);
+
+            String status = transactionReceipt.getStatus();
+            transactionReceipt.setMessage(StatusCode.getStatusMessage(status));
+
+        } catch (Exception e) {
+            transactionReceipt.setStatus(StatusCode.ExceptionCatched);
+            transactionReceipt.setMessage(e.getMessage());
+            transactionReceipt.setOutput("0x");
+        }
+        return transactionReceipt;
+    }
+
+    public void asyncSendTransaction(
+            TransactionSucCallback callback,
+            String contractAddress,
+            String funcName,
+            Type... args) {
+        final Function function =
+                new Function(
+                        funcName,
+                        Arrays.<Type>asList(args),
+                        Collections.<TypeReference<?>>emptyList());
+
+        ExecuteTransaction executeTransaction =
+                new ExecuteTransaction(contractAddress, web3j, credentials, gasPrice, gasLimit);
+
+        executeTransaction.asyncSend(function, callback);
+    }
+
+    public void asyncSendTransaction(
+            TransactionSucCallback callback,
+            BigInteger gasPrice,
+            BigInteger gasLimit,
+            String contractAddress,
+            String funcName,
+            Type... args) {
+        final Function function =
+                new Function(
+                        funcName,
+                        Arrays.<Type>asList(args),
+                        Collections.<TypeReference<?>>emptyList());
+
+        ExecuteTransaction executeTransaction =
+                new ExecuteTransaction(contractAddress, web3j, credentials, gasPrice, gasLimit);
+
+        executeTransaction.asyncSend(function, callback);
+    }
+
+    public List<Type> decode(String data, TypeReference<?>... typeReferences) {
+        if (data.isEmpty() || data.equals("0x")) return null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732356)